components/esp_lvgl_port/esp_lvgl_port.c

```diff
@@ -23,7 +23,7 @@
 #include "esp_lcd_touch.h"
 #endif
 
-#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 1)
+#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(4, 4, 4)
```

Comment:
Unfortunately, the feature with esp-lcd `esp_lcd_panel_io_register_event_callbacks` is not included in v5.0.0. 

So it should work on:  <v4.4.4, v5.0.0) ∪ <v5.0.1, ∞)

Reply:
The CI did not catch it because we test on release _branches_ rather than on release _tags_

Reply:
You are right. Fixed.